applications/MeshingApplication/tests/mmg_lagrangian_test/beam3D_test_result.sol

```diff
@@ -7,173 +7,173 @@ Dimension
-14.7686109193122  0  14.7686109193122  0  0  14.7686109193122  
-16.9554609692555  0  16.9554609692555  0  0  16.9554609692555  
-13.455540017933  0  13.455540017933  0  0  13.455540017933  
-14.185483630148  0  14.185483630148  0  0  14.185483630148  
-15.8277064200399  0  15.8277064200399  0  0  15.8277064200399  
-14.1849599388393  0  14.1849599388393  0  0  14.1849599388393  
-14.1018061112717  0  14.1018061112717  0  0  14.1018061112717  
-13.4442013135134  0  13.4442013135134  0  0  13.4442013135134  
-13.5999714770016  0  13.5999714770016  0  0  13.5999714770016  
-14.5481936084951  0  14.5481936084951  0  0  14.5481936084951  
-14.5479848494158  0  14.5479848494158  0  0  14.5479848494158  
-12.9909304345219  0  12.9909304345219  0  0  12.9909304345219  
-12.8092665161218  0  12.8092665161218  0  0  12.8092665161218  
-11.3983751171588  0  11.3983751171588  0  0  11.3983751171588  
-11.6144472344102  0  11.6144472344102  0  0  11.6144472344102  
-12.9055162943599  0  12.9055162943599  0  0  12.9055162943599  
-11.9333785334854  0  11.9333785334854  0  0  11.9333785334854  
-12.1137609258182  0  12.1137609258182  0  0  12.1137609258182  
-12.5696566494541  0  12.5696566494541  0  0  12.5696566494541  
-11.3550189726419  0  11.3550189726419  0  0  11.3550189726419  
-10.3677942410406  0  10.3677942410406  0  0  10.3677942410406  
-10.1398620246949  0  10.1398620246949  0  0  10.1398620246949  
-10.760157630112  0  10.760157630112  0  0  10.760157630112  
-10.259438150323  0  10.259438150323  0  0  10.259438150323  
-9.65921823604312  0  9.65921823604312  0  0  9.65921823604312  
-9.1418657201665  0  9.1418657201665  0  0  9.1418657201665  
-9.39233794545191  0  9.39233794545191  0  0  9.39233794545191  
-9.02694252618019  0  9.02694252618019  0  0  9.02694252618019  
-9.72916342998384  0  9.72916342998384  0  0  9.72916342998384  
-9.58652539038531  0  9.58652539038531  0  0  9.58652539038531  
-8.32678915384327  0  8.32678915384327  0  0  8.32678915384327  
-9.27454215520464  0  9.27454215520464  0  0  9.27454215520464  
-9.53277137698498  0  9.53277137698498  0  0  9.53277137698498  
-8.89894818733828  0  8.89894818733828  0  0  8.89894818733828  
-8.51660823683054  0  8.51660823683054  0  0  8.51660823683054  
-8.22609130240348  0  8.22609130240348  0  0  8.22609130240348  
-7.87642627781283  0  7.87642627781283  0  0  7.87642627781283  
-8.33949942560789  0  8.33949942560789  0  0  8.33949942560789  
-8.5807679085591  0  8.5807679085591  0  0  8.5807679085591  
-7.85107300473627  0  7.85107300473627  0  0  7.85107300473627  
-8.42153377613511  0  8.42153377613511  0  0  8.42153377613511  
-6.91498232488896  0  6.91498232488896  0  0  6.91498232488896  
-8.02491969678893  0  8.02491969678893  0  0  8.02491969678893  
-7.40582739573172  0  7.40582739573172  0  0  7.40582739573172  
-7.15130340264934  0  7.15130340264934  0  0  7.15130340264934  
+27.7057416888572  0  27.7057416888572  0  0  27.7057416888572  
```

Comment:
why do the results change?
I thought this is only refactoring?

Reply:
If you take a look deeper will see that the values are reordered. I can control the output of the sol files of MMG